### PR TITLE
Added options to command find_package

### PR DESCRIPTION
**Pull request by bmarques1995**
Now is possible to use the command find_package with cmake to find ngtcp2, I've done the same to nghttp3 recently.

### CODE REVIEW

File: lib/config.cmake.in [hunk 1]
```diff
@@ -0,0 +1,3 @@
+include(CMakeFindDependencyMacro)
+
+include("${CMAKE_CURRENT_LIST_DIR}/@NGTCP2_TARGETS_EXPORT_NAME@.cmake")
```
Comment (tatsuhiro-t) [inline lib/config.cmake.in]: Missing line feed at the end of the line.

**Review by tatsuhiro-t — CHANGES_REQUESTED** [review]

**Comment by tatsuhiro-t** [comment]
@vszakats What do you think about this change?

**Comment by vszakats** [comment]
It looks good to me, and on par with the mentioned https://github.com/ngtcp2/nghttp3/pull/228.

File: lib/config.cmake.in (hunk 1 above)
Comment (bmarques1995) [reply lib/config.cmake.in]: done

**Review by tatsuhiro-t — APPROVED** [review]

**Comment by tatsuhiro-t** [comment]
Thank you for PR.  Merged now.

**Pull request merged**